ignite-3/src/jepsen/ignite3/append.clj

```diff
@@ -100,51 +111,87 @@
 
 ; ---------- General scenario ----------
 
-(defn invoke-ops [^Ignite ignite acc ops]
+(defn extract-reason [exc]
```

Comment:
Will this prevent the known exception from being printed to the log?

Reply:
Exactly! Under a significant load, logs are simply polluted by megabytes of similar stacktraces, but this function allows us to collapse most of them into a single keyword. As a result, logs occupy less space, and non-stanard exceptions obtain much higher visibility in them.